packages/apps/plugins/plugin-debug/src/components/DebugPanel.tsx

```diff
@@ -35,6 +39,18 @@ export const DebugPanel: FC<PropsWithChildren<{ menu: ReactNode }>> = ({ menu, c
           </div>
         )}
       </div>
+    </MainOrArticle>
+  );
+};
+
+const MainOrArticle: FC<PropsWithChildren<{ role: 'main' | 'article' }>> = ({ role, children }) => {
+  return role === 'main' ? (
+    <Main.Content classNames={[baseSurface, fixedInsetFlexLayout, topbarBlockPaddingStart, bottombarBlockPaddingEnd]}>
+      {children}
     </Main.Content>
+  ) : (
+    <div role='none' className='row-span-2 rounded-t-md overflow-x-auto'>
```

Comment:
Does `row-span-2` have to do with the toolbar? If so, be sure this works when it’s both enabled and disabled.